website/docs/d/vra_region_enumeration_azure.html.markdown

```diff
@@ -0,0 +1,31 @@
+---layout: "vra"
+page_title: "VMware vRealize Automation: vra_region_enumeration_azure"
+description: |-
+  Provides a data lookup for vra_region_enumeration_azure.
+---
+
+# Data Source: vra_region_enumeration_azure
+## Example Usages
+
+This is an example of how to read a region enumeration data source for Azure cloud account.
+
+**Region enumeration azure data source by its id:**
+```hcl
+data "vra_region_enumeration_azure" "this" {
+  id = vra_region_enumeration_azure.this.id
+}
+```
+## Argument Reference
+
+* `id` - (Optional) The id of the region enumeration for Azure account.
+
+## Attribute Reference
+* `application_id` - Azure Client Application ID
+
+* `application_key` - Azure Client Application Secret Key
+
+* `subscription_id` - Azure Subscribtion ID
+
+* `tenant_id` - Azure Tenant ID
```

Comment:
`application_id`, `application_key`, `subscription_id` and `tenant_id` are `Required` arguments, so should be moved to Argument Reference section.

`id` is attribute. It should be moved to Attribute Reference section.